mater/lib/src/stores/mod.rs

```diff
@@ -6,42 +6,122 @@ pub use blockstore::Blockstore;
 pub use file::FileBlockstore;
 pub use filestore::create_filestore;
 
-/// The default block size, as defined in
-/// [boxo](https://github.com/ipfs/boxo/blob/f4fe8997dcbeb39b3a4842d8f08b34739bfd84a4/chunker/parse.go#L13).
-pub(crate) const DEFAULT_BLOCK_SIZE: usize = 1024 * 256;
+/// The default chunk size for balanced trees (256 KiB)
+/// Reference: https://github.com/ipfs/boxo/blob/f4fe8997dcbeb39b3a4842d8f08b34739bfd84a4/chunker/parse.go#L13
+pub const DEFAULT_CHUNK_SIZE: usize = 256 * 1024;
 
-/// The default tree width, also called links per block, as defined in
-/// [boxo](https://github.com/ipfs/boxo/blob/625ba769263c2beeec934836f54bbd6624db945a/ipld/unixfs/importer/helpers/helpers.go#L16-L30).
-pub(crate) const DEFAULT_TREE_WIDTH: usize = 174;
+/// The default number of children per parent node in balanced trees.
+/// This value comes from the go-ipfs implementation and provides a good balance
+/// between tree depth and width for most use cases.
+pub const DEFAULT_TREE_WIDTH: usize = 174;
 
-/// Store configuration options.
+/// Store configuration options for controlling how data is stored and structured.
+#[derive(Debug, Clone)]
 pub enum Config {
-    /// The store should use the balanced tree layout,
-    /// generating byte chunks of `chunk_size` and
-    /// generating parent nodes every `tree_width` nodes.
+    /// Creates a balanced tree structure by generating fixed-size chunks
+    /// and arranging them into a tree with a specified width.
+    /// By default, content is wrapped in UnixFS format for IPFS compatibility.
     Balanced {
-        /// The size of the byte chunks.
+        /// Size of each chunk in bytes. Defaults to 256 KiB.
+        /// Larger chunks reduce tree depth but increase minimum storage unit size.
         chunk_size: usize,
-        /// The number of children per parent node.
+
+        /// Maximum number of children per parent node. Defaults to 174.
+        /// This affects tree shape and traversal performance.
         tree_width: usize,
+
+        /// If true, store content directly without UnixFS metadata.
+        /// More space efficient but loses IPFS compatibility features.
+        /// Default is false (uses UnixFS wrapping).
+        raw_mode: bool,
     },
 }
 
 impl Config {
-    /// Create a new [`Config::Balanced`].
-    pub fn balanced(chunk_size: usize, tree_width: usize) -> Self {
+    /// Creates a new balanced tree configuration with the specified parameters.
+    ///
+    /// # Arguments
+    /// * `chunk_size` - Size of each data chunk in bytes
+    /// * `tree_width` - Maximum number of children per parent node
+    /// * `raw_mode` - Whether to store content directly without UnixFS wrapping
+    pub fn balanced(chunk_size: usize, tree_width: usize, raw_mode: bool) -> Self {
         Self::Balanced {
             chunk_size,
             tree_width,
+            raw_mode,
         }
     }
+
+    /// Creates a new balanced tree configuration with UnixFS wrapping (recommended).
```

Comment:
This is the place where you explain why using UnixFS is the default instead of just stating it is recommended